python/ql/src/experimental/semmle/python/frameworks/Zip.qll

```diff
@@ -0,0 +1,29 @@
+private import python
+private import experimental.semmle.python.Concepts
+private import semmle.python.dataflow.new.DataFlow
+private import semmle.python.ApiGraphs
+
+private module Zip {
+  private class CopyFiles extends DataFlow::CallCfgNode, CopyFile::Range {
+    CopyFiles() {
+      this =
+        API::moduleImport("shutil")
+            .getMember(["copyfile", "copy", "copy2", "copytree", "move"])
+            .getACall()
```

Comment:
it can be vulnerable, and the code below can explain.
https://github.com/ahmed-farid-dev/codeql/blob/d89ed8b98be25f53c88fafb2d9c4c6f972c3f442/python/ql/test/experimental/query-tests/Security/CWE-022/zipslip_bad.py#L25-L30

Reply:
for  the autoformat checks, i use <code>codeql query format</code>